src/playbacks/hls/hls.js

```diff
@@ -119,6 +134,25 @@ export default class HLS extends HTML5VideoPlayback {
     this.trigger(Events.PLAYBACK_SETTINGSUPDATE)
   }
 
+  onError(evt, data) {
+    if (data && data.fatal && this.recoverAttemptsRemaining > 0) {
+      this.recoverAttemptsRemaining -= 1
+      switch (data.type) {
+        case HLSJS.ErrorTypes.NETWORK_ERROR:
+          Log.warning(`hlsjs: trying to recover from network error, evt ${evt}, data ${data} `)
+          this.hls.startLoad()
+          break
+        case HLSJS.ErrorTypes.MEDIA_ERROR:
+          Log.warning(`hlsjs: trying to recover from media error, evt ${evt}, data ${data} `)
+          this.recover()
+          break
+        default:
+          Log.error(`hlsjs: could not recover from error, evt ${evt}, data ${data} `)
```

Comment:
@ronag no problem, usually when we face an error that we can't do nothing we trigger a playback error therefore the player can show something for the final user. 

The [tag video](https://github.com/clappr/clappr/blob/master/src/playbacks/html5_video/html5_video.js#L291), for instance, it handles the error this way:
```javascript
  onError(event) {
    this.trigger(Events.PLAYBACK_ERROR, this.el.error, this.name)
  }
```
In our case we can trigger an even like this but changing the reason :)